data/advisor.yml

```diff
@@ -3,6 +3,18 @@ advisor:
   color: 2
   features:
     _scope: '[data-ouia-app-id="advisor"]'
+    "Advisor - Card: Pathways": 
+      selectors:
+        - '{}[data-testid="Pathways"]'
```

Comment:
I think you need a space between the {} and the rest of the selector. Otherwise looks good.
Same for all of them

Reply:
but the rest of the existing ones do not have that space. Or am I misunderstanding the position of the space

Reply:
They should all have the space

https://github.com/RedHatInsights/pendo-config-automation/blob/main/docs/data.md#scope